Remove dual dispatchers docs and CHANGELOG (#1177)
OpenZeppelin/cairo-contracts: docs/modules/ROOT/pages/interfaces.adoc

```

=== Dispatcher traits

> Traits decorated with `#[starknet::interface]` automatically generate a dispatcher that can be used to interact with contracts that implement the given interface. They can be imported by appending the `Dispatcher` and `DispatcherTrait` suffixes to the trait name, like this:

Author: Added a mention and a link.

```

=== Dispatcher traits

:interacting-with-another-contract: https://book.cairo-lang.org/ch15-02-interacting-with-another-contract.html[Interacting with another contract]

Traits annotated with `#[starknet::interface]` automatically generate a dispatcher that can be used to interact with contracts that implement the given interface. They can be imported by appending the `Dispatcher` and `DispatcherTrait` suffixes to the trait name, like this:

```cairo
use openzeppelin_token::erc20::interface::{IERC20Dispatcher, IERC20DispatcherTrait};
```

Other types of dispatchers are also auto-generated from the annotated trait. See the
{interacting-with-another-contract} section of the Cairo book for more information.

NOTE: In the example, the `IERC20Dispatcher` is the one used to interact with contracts, but the
`IERC20DispatcherTrait` needs to be in scope for the functions to be available.

== Dual interfaces

> WARNING: `camelCase` functions are deprecated and maintained only for Backwards Compatibility. We recommend to
> integrate you contracts and components with the `snake_case` interfaces. The `camelCase` functions will be removed in

Reviewer: ```suggestion
WARNING: `camelCase` functions are deprecated and maintained only for Backwards Compatibility.
It's recommended to only use `snake_case` interfaces with contracts and components. The `camelCase` functions will be removed in
```
